napari/utils/_tests/test_translations.py

```diff
@@ -0,0 +1,174 @@
+"""Test the translations API."""
+
+import subprocess
+import sys
+from pathlib import Path
+
+import pytest
+
+from ...utils import translations
```

Comment:
Our convention has been to use absolute imports in our tests, do you mind changing here and below

Reply:
Will do!